tests/test_remaining_graph_rules.py

```
@@ -1,12 +1,13 @@
 """Unit and integration tests for Phase 2J+K GraphRules.
 
-Covers L056, R401, collection metadata rules (L087, L088, L096, L103–L105),
-and plugin/schema rules (L089, L090, L095).
+Covers L056, R401, R402, collection metadata rules (L087, L088,
+L096, L103–L105), and plugin/schema rules (L089, L090, L095).
```

Ruff (0.16.2)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)